rototiller.gemspec

```diff
@@ -17,6 +17,8 @@ Gem::Specification.new do |s|
   #Development dependencies
   s.add_development_dependency 'rspec', '~> 3.1.0'
   s.add_development_dependency 'simplecov'
+  s.add_development_dependency 'beaker'
```

Comment:
this is in the :system_tests group in the Gemfile so it can be excluded upon bundle install if one wants. 
do we think it's a development dependency?  maybe
whatever we decide, same should be true of hostgen, i suppose